README.md

```diff
@@ -1,5 +1,6 @@
 # Create Checkbox and Radio Button Components in a Blazor Server Application
```

Comment:
The topic is for both CheckBox and Radio Button but we are describing only Radio Button. Change it with proper content.